apps/liteprotocoltester/.env

```diff
@@ -4,7 +4,7 @@ START_PUBLISHING_AFTER=10
 NUM_MESSAGES=0
 # 0 for infinite number of messages
 
-DELAY_MESSAGES=8000
+DELAY_MESSAGES=188
```

Comment:
I'm curious as to why this figure :D

Reply:
Ah, I see... it remained from my last waku-sim run.
While 8000ms is set for TWN... not to harm rln limit... on waku-sim I needed to stress the network... hence the much lower value.
I'd better leave the (my) .env file here for an example.